### PR TITLE
Update copied event reminders to link to new event when creating from…

### PR DESCRIPTION
**Pull request by ufundo**
… template using api4

Overview
----------------------------------------
A fix for https://lab.civicrm.org/dev/core/-/issues/3183

Comments
----------------------------------------
The code around this is very hard to follow so wasn't able to get my head around how it was supposed to work... hence this somewhat inelegant approach.

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
style warning.. - see https://test.civicrm.org/job/CiviCRM-Core-PR/48118/checkstyle/

**Comment by ufundo** [comment]
fixed the style issues 👍

**Comment by eileenmcnaughton** [comment]
@agileware-justin @agileware-fj  - I feel like you use Event Templates & might be able to comment - note @ufundo is a relatively new contributor so I was going to go easy on the 'test please' bit :-)

**Comment by agileware-justin** [comment]
@eileenmcnaughton if you or someone else could help with https://github.com/civicrm/civicrm-core/pull/23098#issuecomment-1111583999 - then sure happy to review this one.

**Comment by agileware-justin** [comment]
Thanks E. Testing now...

**Comment by agileware-justin** [comment]
@ufundo good work, tested and confirmed working.

@eileenmcnaughton ready to roll.

It works, oh wait...

**Comment by eileenmcnaughton** [comment]
thanks @agileware-justin !
